client/extensions/wp-super-cache/components/debug/index.jsx

```diff
@@ -95,8 +98,15 @@ class DebugTab extends Component {
 						</FormFieldset>
 						<p>
 							{ translate(
-								'Currently logging to: %(location)s',
-								{ args: { location: cache_path + wp_cache_debug_log } }
+								'Currently logging to: {{ExternalLink}}%(location)s{{/ExternalLink}}',
+								{
+									args: { location: cache_path + cacheFilename },
+									components: {
+										ExternalLink: <ExternalLink
```

Comment:
There's a dedicated prop needed to enable it. It looked bad when inlined into that sentence, so I dropped it.